src/components/pages/manage/productions/index/index.js

```diff
@@ -26,6 +27,8 @@ import { Productions } from './Productions';
 
 const productionsPerPage = 15;
 
+const productionsQueryKey = ['productions', { limit: 15, name: '', start: 0 }];
```

Comment:
@brampauwelyn have you tried the key 'productions' after you fixed it? I think it should work

Reply:
Nope I didn't :p 
can you try it?

Reply:
I tried it quickly, seems like you have to pass the whole key